maintenance_curation.pt.Rmd

```diff
@@ -6,213 +6,215 @@ aliases:
+  é uma lista centralizada dos pacotes R que são mantidos atualmente (ou que foram
+  mantidos anteriormente) pela rOpenSci.
+  Ele contém metadados essenciais sobre os pacotes, incluindo o status de desenvolvimento e de revisão,
+  e será a fonte de dados para exibição em sites, *badges*, etc. Ele permite
+  que essa lista seja mantida de forma independente do pacote ou das
   plataformas de hospedagem de infraestrutura.
 
 ## Pacotes mantidos pela equipe {#staff-maintained-packages}
 
-Os pacotes mantidos pela equipe são desenvolvidos e mantidos pela equipe da rOpenSci
-como parte dos projetos rOpenSci. Esses pacotes também podem ser revisados por pares
-mas não são necessariamente revisados por pares. Muitos são de infraestrutura
-que estão fora do escopo da revisão por pares.
+Os pacotes mantidos pela equipe são pacotes desenvolvidos e mantidos pela equipe da rOpenSci
+como parte dos projetos internos da rOpenSci. Esses pacotes também podem ser revisados por pares
+mas não são necessariamente revisados por pares. Muitos desses pacotes
+estão fora do escopo da revisão por pares.
 
 - Os pacotes mantidos pela equipe serão listados no registro com a tag
-  "staff\_maintained" e listados na página da Web de pacotes da rOpenSci ou similar
-  locais com a tag "staff-maintained" (mantido pela equipe)
+  "staff\_maintained" e listados na página da Web de pacotes da rOpenSci, ou em locais similares
+  com a tag "staff-maintained" (mantido pela equipe)
 
-- Esses pacotes serão armazenados no GitHub "ropensci"
-  organização
+- Esses pacotes serão armazenados no dentro da organização no GitHub
+  chamada "ropensci"
+  
 
-- Os pacotes mantidos pela equipe e seus documentos serão criados pela rOpenSci
-  [sistema](https://status.ropensci.org/). Esse sistema não envia notificações
-  mas gera resultados como status de commit do GitHub (marca de seleção vermelha ou cruz vermelha).
+- Os pacotes mantidos pela equipe e seus documentos serão criados pelo [sistema](https://status.ropensci.org/)
+  da rOpenSci. Esse sistema não envia notificações,
+  mas gera resultados como status de commit do GitHub (o *red check mark* ou o *red cross*).
 
 - Quando os pacotes falham nas verificações, a equipe da rOpenSci se esforça para corrigir
-  alterações, priorizando os pacotes com base na base de usuários (downloads),
-  dependências reversas ou objetivos estratégicos.
+  as alterações, priorizando os pacotes com base no volume de usuários (isto é, o volume de *downloads*), de
+  dependências reversas ou de objetivos estratégicos.
 
 - Em uma base semestral ou anual, a rOpenSci analisará todos os pacotes
-  que estão falhando há mais de um mês para determinar se você deve transferi-los para o [ organização "ropensci-archive" do GitHub](https://github.com/ropensci-archive).
+  que estão falhando há mais de um mês para determinar se você deve
+  transferi-los para a [ organização "ropensci-archive" do GitHub](https://github.com/ropensci-archive).
```

Comment:
I'm suggesting "no GitHub" instead of "do GitHub" to avoid sounding like it is a official organization by GitHub. "no GitHub" clarifies that is it hosted by GitHub.

```suggestion
  transferi-los para a [organização "ropensci-archive" no GitHub](https://github.com/ropensci-archive).
```